.github/workflows/esphome_build.yaml

```
@@ -23,19 +23,21 @@ jobs:
+          sed -i 's/YOUR_NEXXTENDER_MAC_ADDRESS/00:1A:7D:DA:71:13/' secrets.yaml
+          sed -i 's/YOUR_NEXXTENDER_PASSKEY/123456/' secrets.yaml
           echo "Secrets template moved."
-        working-directory: ${{ github.workspace }}
+        working-directory: config
 
       - name: Build ESPHOME az-delivery-devkit-v4
         uses: esphome/build-action@v6
         with:
-          yaml-file: config/nexxtender.yaml
+          yaml-file: nexxtender.yaml
+        working-directory: config
```

actionlint (1.7.4)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)